docs/content/configuration/csi_ug_config_create_replication.md

```diff
@@ -0,0 +1,46 @@
+# Creating a VolumeReplication
+
+Create a VolumeReplication yaml file to replicate a specific PersistentVolumeClaim (PVC).
+
+VolumeReplicationClass needs to be present before a VolumeSnapshot can be created. For more information, see [Creating a VolumeReplicationClass](csi_ug_config_create_vol_replicationclass.md).
+
+**Note:** Replication is also known as mirroring in many storage systems.
+
+When replicating a volume, be sure to follow all of the replication configurations, found in [Compatibility and requirements](../installation/csi_ug_requirements.md) before snapshot creation.
+
+1.  Replicate a specific  PersistentVolumeClaim (PVC) using the demo-replication.yaml.
```

Comment:
```suggestion
1.  Replicate a specific PersistentVolumeClaim (PVC) using the demo-replication.yaml.
```
is there an editor you could use which helps avoid superfluous spaces?

Reply:
DONE
I've tried to go through with a search to see which spaces can be removed.
I am using the best editor that I have available currently to work with the MD files, github, and use the other one ID tool that I have at my disposal for this to help with style, grammar, word usage etc. according to IBM standards. I use it as I can and it usually catches these types of issues.

Reply:
I still see this double space here

Reply:
@oriyarde  Please check now. I might have not pushed the changes fully earlier.